homeassistant/components/sensibo/services.yaml

```diff
@@ -44,3 +44,56 @@ timer:
           min: 0
           step: 1
           mode: box
+pure_boost:
+  name: Pure Boost
+  description: Configure Pure Boost settings.
+  target:
+    entity:
+      integration: sensibo
+      domain: climate
+  fields:
+    state:
+      name: State
+      description: State to set.
+      required: true
+      example: true
+      selector:
+        boolean:
```

Comment:
Maybe call this `enabled`?  It took me a bit to figure out what it did.


<img width="1197" alt="Screen Shot 2022-06-18 at 13 31 33" src="https://user-images.githubusercontent.com/663432/174451879-880496ae-6ec2-4c29-aa29-044612733da5.png">

Reply:
Thinking about this some more, it might make more sense to have one service to enable, and one service to disable.

Reply:
If we go with how Sensibo themselves does it they have Enable/Disable as a button and then "Configure Pure Boost" in settings (which also turns enable Pure Boost if currently disabled.
That would be in total 3 services but I think that's stretching it so I would also think your suggestion with enable/disable service is the better way. `enable` also means to configure boost settings but `disable` purely means disable it.

Reply:
I think you could still do it with two services by making the args optional for the enable service.

Reply:
Probably the best so now it's configured as such and works well on my device.